cloud_Azure/terraform/module/examples/multiple_accounts_multiple_resource_group/profiles_populator.py

```diff
@@ -0,0 +1,428 @@
+import argparse
+import configparser
+from dataclasses import dataclass
+import logging
+import os
+import shutil
+import sys
+from datetime import datetime
+from itertools import islice
+from pathlib import Path
+from typing import Any, List, Optional
+
+from azure_cli import az_cli
+
+from profiles import AzureProfile, load_profiles
+
+
+# redirect logs to file so they don't mess up user-program interaction
+log = logging.getLogger(__name__)
+logging.basicConfig(
+    format="%(asctime)s [%(name)s] %(levelname)s: %(message)s", filename="populator.log", level=logging.INFO
```

Comment:
How would user find out where is the log goes without reading the code? For command-line utilities it is generally better to log  `stderr` (which the `logging` modules does by default). Obviously, you need to avoid logging things that are printed to stdout anyway.

Reply:
I can log to stderr but then the user needs to redirect stderr to file, otherwise the console will get cluttered. My approach separates user-machine interactions from logger printouts targeted for troubleshooting

Reply:
Please, log to stderr, but limit log messages (by default) only to information that the user really needs to see. Redirecting stderr is not hard, but if you want, you can add an option allowing users to specify name of the log file.

Reply:
I will modify logging in separate commit